seshdash/views.py

```diff
@@ -700,3 +720,37 @@ def historical_data(request):
         context_dict['sort_dict'] = sort_data_dict
         return render(request, 'seshdash/historical-data.html', context_dict);
 
+
+#function for Graph Generations   
+@login_required
+def graphs(request):
+    if request.method == 'POST':
+        results = {}
+        data_values =[]
+        date_time = []
+        time = request.POST.get('time','')
+        choices = request.POST.getlist('choice[]')
+        active_id = request.POST.get('active_site_id','')
+        active_site = Sesh_Site.objects.filter(id=active_id)
+        active_site_name = active_site [0]
+        current_site = active_site_name.site_name
+        for choice in choices:
+            data_values = []
+            time_delta_dict = {'24h':{'hours':24},'7d':{'days':7},'30d':{'days':30}}
```

Comment:
any constant containing dictionaries should be declared in the beginning of the function